content/en/docs/contribute/new-content/new-features.md

```diff
@@ -147,14 +157,34 @@ feature gates). Watch out for language such as ”The `exampleSetting` field
 is a beta field and disabled by default. You can enable it by enabling the
 `ProcessExampleThings` feature gate.”
 
-If your feature is GA'ed or deprecated, make sure to move it from the
+If your feature is GA'ed or deprecated, 
+include an additional `stage` entry within the `stages` block in the description file. 
+Ensure that the Alpha and Beta stages remain intact. 
+This step transitions the feature gate from the 
 [Feature gates for Alpha/Feature](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) table
-to [Feature gates for graduated or deprecated features](/docs/reference/command-line-tools-reference/feature-gates-removed/#feature-gates-that-are-removed)
-table with Alpha and Beta history intact.
-
-Eventually, Kubernetes will stop including the feature gate at all.
-In that case, you move it from [Feature gates for graduated or deprecated features](/docs/reference/command-line-tools-reference/feature-gates-removed/#feature-gates-that-are-removed)
-and into a separate page, [Feature Gates (removed)](/docs/reference/command-line-tools-reference/feature-gates-removed/).
+to [Feature gates for graduated or deprecated features](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features) table. For example:
+
+```yaml
```

Comment:
Using a `diff` block might be useful?

Reply:
Now I've incorporated the `{{< highlight >>}}` shortcode to emphasize the added block.